### PR TITLE
Implement API facility to move tables

### LINKED ISSUE #718: Implement API facilitiy to release tables
This means to move a table from one schema to another, including the following steps:

1. Update the schema in the table registry (`dataedit.models.Table`)
2. Update schemas in table that binds tags to tables (`dataedit.structures.TableTags`)
3. Move table in the oedb using sqlalchemy

The usual use-case is moving a table from model_draft to any of the other schemas (excluding sandbox).
Only white-listed schemas should be available.

### PR DESCRIPTION
**Pull request by MGlauer**
This implements a rudimentary method to move tables including respective meta tables and tags and permissions

Closes #718

### CODE REVIEW

**Comment by MGlauer** [comment]
POST: `/api/v0/schema/{from_schema}/tables/{table}/move/{to_schema}/`

**Comment by Ludee** [comment]
I didn't test the function yet but I would suggest minor changes in the error messages:

~~"Table register not found"~~ "Table for schema movement not found"
~~"No schema register found"~~ "Target schema not found"
~~"Invalid schema"~~ "Invalid origin or target schema"

File: api/actions.py [hunk 1]
```diff
@@ -1370,6 +1370,46 @@ def clear_dict(d):
...
+
+    table = read_pgid(table)
+    engine = _get_engine()
+    Session = sessionmaker(engine)
+    session = Session()
+    try:
+        try:
+            t = DBTable.objects.get(name=table, schema__name=from_schema)
+        except DBTable.DoesNotExist:
+            raise APIError("Table register not found")
```
Comment (christian-rli) [inline api/actions.py:1383]:
```suggestion
            raise APIError("Table for schema movement not found")
```

**Review by christian-rli — APPROVED** [review]
@MGlauer sorry for the late review. I thought this was already implemented. I went ahead and commited @Ludee 's suggestions and I'm looking forward to using the functionality. Will I have to wait for the next release for it to work?

**Pull request merged**